devito/arguments.py

```diff
@@ -85,6 +91,44 @@ def verify(self, value):
         return self._value is not None
 
 
+class ConstantArgument(Argument):
+
+    """ Class representing scalar arguments that a kernel might expect.
```

Comment:
Shouldn't then this inherit from the ``ScalarArgument``, thus reusing some of its code (e.g, ``def dtype``)